help/en/releasenotes/current-draft-note.shtml

```diff
@@ -201,12 +201,12 @@
 
         <h4>Digitrax</h4>
             <ul>
-                <li></li>
+              <li>Added Kato EM13 in Digitrax family: newer versions of this decoder have a Digitrax manufacturer code and a full fledged Digitrax SW.</li>
             </ul>
 
         <h4>Doehler &amp; Haas</h4>
             <ul>
-                <li></li>
+              <li>Added PD10MU: This is a specific decoder for <a href=https://www.dm-toys.de/fr/produit-detail/DH_PD10MU-4.html target=_BLANK>ModellBahn Union / DM-Toys</a>. Same SW as PD12A, but N scale form factor.</li>
```

Comment:
I think it's good if the URL in the A tag is quoted:
`<a href="https://www.dm-toys.de/fr/produit-detail/DH_PD10MU-4.html" target="_BLANK">ModellBahn Union / DM-Toys</a>`

I don't know if it's needed, but I have always seen `href` quoted.

Reply:
Works without quotes.